src/stratigraphy/groundwater/groundwater_extraction.py

```diff
@@ -2,12 +2,16 @@
 
 import abc
 import logging
+import math
+import os
 from dataclasses import dataclass
 from datetime import date as dt
 from datetime import datetime
+from pathlib import Path
 
 import fitz
 import numpy as np
+import skimage as ski
```

Comment:
Move all logic related to template matching (including this import) to a separate Python file, and only import this file when template matching is enabled.

Reply:
Done.